posthog/temporal/workflows/s3_batch_export.py

```diff
@@ -40,6 +42,7 @@ class S3InsertInputs:
     data_interval_end: str
     aws_access_key_id: str | None = None
     aws_secret_access_key: str | None = None
+    compression: Literal["gzip", "none"] = "none"
```

Comment:
Not a huge fan of this as I can imagine a stack trace that ends in something like `ValueError: "none"` and being confused about `"none"`. Not sure if the better alternative is `None` or an enum.

Reply:
Have used an enum